### DISCUSSION TITLE
How to use it with virtual lists

### DISCUSSION THREAD

**Discussion by A-kirami** [KingSora/OverlayScrollbars#616] [Q&A] [answered]
Hello!
I am using this library with a virtual list, but I am having a bit of a problem.
The virtual list I used before was https://github.com/Akryum/vue-virtual-scroller, which worked very well with OverlayScrollbars.
But after I changed to https://github.com/reactjser/vue3-virtual-scroll-list, the events of the virtual list could not be triggered.
What should I do to make them work together?

**Comment by KingSora** [comment]
@A-kirami Good day :)

Please create a example of how use the plugin together with overlayscrollbars on StackBlitz / CodeSandbox. - I'll take a look what the problem could be then

**Reply by A-kirami** [reply]
Thank you, I have created [a sample](https://codesandbox.io/p/devbox/objective-phoebe-t6cx46?workspaceId=b2fd163a-c4eb-4e69-a30e-1c5e368184e0).

**Comment by KingSora** [comment, marked as answer]
@A-kirami Alrighty.. I've created an example for you here: https://codesandbox.io/p/devbox/angry-farrell-gk9tg2?file=%2Fsrc%2Fcomponents%2FMessageList.vue%3A29%2C16 

Instead of initializing it always like this you should create a new component which does the initialization for you :)

**Reply by A-kirami** [reply]
This has solved my problem, thank you for your help, and wish you a happy life!
